Source/Core/VideoBackends/OGL/TextureCache.h

```diff
@@ -8,54 +8,45 @@
 
 #include "Common/CommonTypes.h"
 #include "Common/GL/GLUtil.h"
+#include "VideoBackends/OGL/ProgramShaderCache.h"
 
 #include "VideoCommon/TextureCacheBase.h"
 #include "VideoCommon/VideoCommon.h"
 
+class AbstractTexture;
+struct TextureConfig;
+
 namespace OGL
 {
+
+// TODO: Don't do this
```

Comment:
I would really hope this would be resolved before trying to merge this. Otherwise the `s_` is wrong in this case, and this introduces a tremendous amount of global state.